src/platforms/salesforce/actions/list-view-results/find.ts

```diff
@@ -10,7 +11,7 @@ export default action(
     mutation: false,
     schema: z.object({
       id: z.string(),
-      objectType: z.string(),
+      objectType: salesforceSupportedObjectType,
     }),
```

Comment:
Should the typing be tighter?

Reply:
Tighter in the sense only allowing Leads and Contacts? They support listviews for all objects (except LitsView, of course)

Reply:
ah ic perfect